Tests/AppTests/SearchTests.swift

```diff
@@ -625,7 +635,78 @@ class SearchTests: AppTestCase {
                            summary: ""))
         ])
     }
-
+    
+    func test_search_withFilter_stars() throws {
+        // Setup
+        let p1 = Package(id: .id1, url: "1", score: 10)
+        let p2 = Package(id: .id2, url: "2", score: 20)
+        try [p1, p2].save(on: app.db).wait()
+        try Repository(package: p1,
+                       defaultBranch: "main",
+                       name: "1",
+                       owner: "bar",
+                       stars: 50,
+                       summary: "test package").save(on: app.db).wait()
+        try Repository(package: p2,
+                       defaultBranch: "main",
+                       name: "2",
+                       owner: "foo",
+                       stars: 10,
+                       summary: "test package").save(on: app.db).wait()
+        try Version(package: p1, packageName: "p1", reference: .branch("main"))
+            .save(on: app.db).wait()
+        try Version(package: p2, packageName: "p2", reference: .branch("main"))
+            .save(on: app.db).wait()
+        try Search.refresh(on: app.db).wait()
+        
+        do { // Baseline
+            let res = try Search.fetch(app.db, ["test"], page: 1, pageSize: 20).wait()
+            XCTAssertEqual(res.results.count, 2)
+            XCTAssertTrue(res.results.allSatisfy { $0.isPackage })
```

Comment:
Since there's neither a `test` keyword nor a match for the owner this would succeed even if we were including keyword and author results for filtered searches, I think.

Reply:
You're not wrong, and this all went through my thought process. This check is mostly just validating that it's picked up both of the test packages, showing that with no filters it's picking up everything (as expected). I added this particular line as just a sanity check that we somehow didn't completely mess up and include something else 😂 

I suppose I also could've just done the package name check on both packages like I did for the other 4 checks underneath - would that be less confusing?